go.mod

```diff
@@ -14,25 +14,25 @@ require (
 	github.com/jhump/protoreflect v1.13.1-0.20220928232736-101791cb1b4c
```

Comment:
FYI - running `make updategodeps` changed this to `v1.13.0`, which then caused failing tests. If we need to pin a specific version, I think we should do this in the makefile (or perhaps tag a newer release to be pulled in).